ArduPlane/quadplane.cpp

```diff
@@ -1239,6 +1239,8 @@ void QuadPlane::update_transition(void)
         plane.control_mode == TRAINING) {
         // in manual modes quad motors are always off
         if (!tilt.motors_active && !is_tailsitter()) {
+            // set relax multirotor angle and rate loops and zero I terms
+            attitude_control->relax_attitude_controllers();
```

Comment:
Shouldn't this also be done in the TRANSITION_DONE state?

Reply:
this part isn't needed any more, but the rest is